.shippable.yml

```diff
@@ -36,6 +36,7 @@ build:
     - mkdir -p $HOME/odp-shmdir
     - ./bootstrap
     - if [ "${CC#clang}" != "${CC}" ] ; then export CXX="${CC/clang/clang++}"; fi
+    - if [ "${CC#clang}" != "${CC}" ] ; then export CFLAGS="-fPIC"; fi
```

Comment:
NAK, this hides the issue instead of fixing it.

Reply:
yep, that is why patches are WIP.  Will wait for Shippable tests results.